### PR TITLE
add check to ensure that at least one multiple choice option has text

### PR DESCRIPTION
**Pull request by IvanYu327**
## Notion ticket link

[[Camp Creation] Require at least one option for checkbox and multiple choice form questions](https://www.notion.so/uwblueprintexecs/8902e20b0cae4db2b4776259238f8a4e?v=4cf916ddabfc4438a2685c0990a4fc57&p=84664a4b12fa41fab32d4c81b0af2775&pm=c)

## Implementation description
* Add isQuestionOptionsInvalid to check that the multiple choice options are valid

## Steps to test
1. Go to add camp and try to save multiple choice questions with no options

## What should reviewers focus on?
* `AddQuestionModal.tsx`

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by twilkhoo** [comment]
Looks like there are some linting issues right now

File: frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx [hunk 1]
```diff
@@ -102,6 +108,16 @@ const AddQuestionModal = ({
       return;
     }
 
+    console.log(questionOptions);
```
Comment (twilkhoo) [inline frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx:111]: Delete this

Comment (IvanYu327) [reply frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx:111]: oops

**Review by twilkhoo — APPROVED** [review]
LGTM

**Pull request merged**